norfair/video.py

```diff
@@ -195,3 +200,60 @@ def abbreviate_description(self, description):
                 description[: space_for_description // 2 - 3],
                 description[-space_for_description // 2 + 3 :],
             )
+
+
+class VideoFromFrames:
+    def __init__(self, input_path, save_path=".", information_file=None):
+
+        if information_file is None:
+            information_file = metrics.InformationFile(
+                file_path=os.path.join(input_path, "seqinfo.ini")
+            )
+
+        file_name = os.path.split(input_path)[1]
+
+        # Search framerate on seqinfo.ini
+        fps = information_file.search(variable_name="frameRate")
+
+        # Search resolution in seqinfo.ini
+        horizontal_resolution = information_file.search(variable_name="imWidth")
+        vertical_resolution = information_file.search(variable_name="imHeight")
+        image_size = (horizontal_resolution, vertical_resolution)
+
+        # Search total frames in seqinfo.ini
+        self.length = information_file.search(variable_name="seqLength")
+
+        videos_folder = os.path.join(save_path, "videos")
+        if not os.path.exists(videos_folder):
+            os.makedirs(videos_folder)
+
+        video_path = os.path.join(videos_folder, file_name + ".mp4")
+        fourcc = cv2.VideoWriter_fourcc(*"mp4v")
+
+        self.file_name = file_name
+        self.input_path = input_path
+        self.frame_number = 1
+        self.video = cv2.VideoWriter(video_path, fourcc, fps, image_size)  # Video file
+
+    def __iter__(self):
+        self.frame_number = 1
+
+    def __next__(self):
+        if self.frame_number <= self.length:
+        frame_path = os.path.join(
+            self.input_path,
+            "img1",
+            str(self.frame_number).zfill(6) + ".jpg",
+        )
+            self.frame_number += 1
+
+        return cv2.imread(frame_path)
+        raise StopIteration()
```

Comment:
This code shouldn't be able to run. The if statements body is not indented, and the raise statement will never execute, did you try to run this?